contribs/gmf/src/directives/search.js

```diff
@@ -228,6 +234,13 @@ gmf.SearchController = function($scope, $compile, $timeout, gettextCatalog,
    */
   this.featureOverlay_ = ngeoFeatureOverlayMgr.getFeatureOverlay();
 
+  var featureStyle = this.scope_['featuresStyle'];
+  if (featureStyle instanceof ol.style.Style ||
+      featureStyle instanceof Array ||
+      featureStyle instanceof Function) {
+    this.featureOverlay_.setStyle(featureStyle);
```

Comment:
The style is meant to be changed by the user, this setter doesn't handle this case.